include/micm/solver/state.hpp

```diff
@@ -59,6 +59,10 @@ namespace micm
     std::size_t number_of_grid_cells_;
     std::unique_ptr<TemporaryVariables> temporary_variables_;
 
+    /// @brief Constructor with parameters
+    /// @param parameters State dimension information
+    State(const StateParameters& parameters);
+
```

Comment:
Can't find the definition of this constructor. Is it a placeholder?

Reply:
it's in the inl file.

Reply:
ahh my bad